packages/components/src/Button/button.stories.tsx

```diff
@@ -101,3 +139,79 @@ withFakeClassName.args = {
   variant: "outline",
   className: "fake-className",
 };
+
+// Show grids with all variants
+
+const gridParameters = {
+  axe: {
+    skip: true,
+  },
+  snapshot: {
+    skip: true,
+  },
+};
+
+const renderSize = (
+  size: ButtonProps["size"],
+  textColor: "white" | "neutral",
+  children: React.ReactNode,
+) =>
+  variants.map((variant) => {
+    // For now, the UI kit only includes alert & brand "flat" buttons
+    const colors = variant === "flat" ? ["alert", "brand"] : allColors;
+
+    return (
+      <React.Fragment key={variant}>
+        <Heading size="h2" color={textColor}>
+          {variant} - {size}
+        </Heading>
+        <table className={styles["variant"]}>
+          <tbody>
+            {states.map((state) => (
+              <tr key={state}>
+                <th scope="row">
+                  <Text size="body" color={textColor}>
+                    {state}
+                  </Text>
+                </th>
+                {colors.map((color) => (
+                  <td key={color} className={styles["color"]}>
+                    <Clickable
```

Comment:
I think I asked this before, but why use Clickable instead of a button?

I may not understand what Clickable is, but I thought it was the base for button and links, but folks should import one of those.

Reply:
oops I didn't even notice that's what was there earlier. idk why this wouldn't be a button either, will update

Reply:
ah I see -- it's because the `state` prop isn't allowed on `Button`, but we added it to `Clickable` just for storybook purposes. I wanted to restructure Clickable as a `createClickable` factory at some point, but for now I left a comment here